kaos/algorithm/visibility_finder.py

```diff
@@ -270,7 +294,9 @@ def determine_visibility(self, error=0.1, tolerance_ratio=0.1, max_iter=100):
         # The subinterval_end is set to start the initial loop iteration
         subinterval_end = start_time
         # Defines the length of the initial subinterval (h)
-        prev_time_step = 1000
+        prev_time_step = 100
```

Comment:
ditto above

Reply:
This one is to match the paper numbers. As you know it has pretty much no effect after the first few interval updates.

Reply:
Oh shit derp